projects/control-service/projects/helm_charts/pipelines-control-service/templates/alertmanager.yaml

```diff
@@ -0,0 +1,53 @@
+{{- /*
+  Copyright 2021-2023 VMware, Inc.
+  SPDX-License-Identifier: Apache-2.0
+ */}}
+
+{{- if .Values.alertmanager.enabled }}
+apiVersion: v1
+kind: List
+metadata:
+  resourceVersion: ""
+items:
+  - apiVersion: monitoring.coreos.com/v1alpha1
+    kind: AlertmanagerConfig
+    metadata:
+      annotations:
+        kubectl.kubernetes.io/last-applied-: ""
+      generation: 6
+      labels:
+        role: infra
```

Comment:
This must be configurable. This is very specific to the deployment of the Alert Manager being deployed in organization. It's used (If I recall correctly) by Alert manager controller to find the correct config.

All labels should be configurable.